cmd/thanos/bucket.go

```diff
@@ -291,6 +299,123 @@ func registerBucketInspect(m map[string]setupFunc, root *kingpin.CmdClause, name
 	}
 }
 
+// RegisterBucketWeb exposes a web interface for the state of Thanos remote store just like `pprof web`
+//
+// TODOS:
+//
+// 1. Some download code is duplicated from `ls -o json`; refactor as needed
+// 2. Some web code is duplicated from query UI, refactor as needed
+// 3. Do we need any of the route prefix?
```

Comment:
Let's drop it, if we will need we will get a separate use case (and hopefully PR) from users.